docs/en/developers/set-autoscale.md

```diff
@@ -233,3 +124,143 @@ Introduce how to automatically scale workloads by cron in Appfile.
   ```
   
   Wait after the period ends, the replicas will be one eventually.
+
+## Setting auto-scaling policy of CPU resource utilization
+Introduce how to automatically scale workloads by CPU resource utilization.
+
+- Prepare Appfile
+  ```yaml
+  name: testapp
+
+  services:
+    express-server:
+      # this image will be used in both build and deploy steps
+      image: zzxwill/kubevela-appfile-demo:v1
+
+      build:
+        # Here more runtime specific build templates will be supported, like NodeJS, Go, Python, Ruby.
+        docker:
+          file: Dockerfile
+          context: .
+
+      cmd: ["node", "server.js"]
+      port: 8080
```

Comment:
We do not need to specify CPU limit?

Reply:
Oh, sorry. I missed it and I didn't make an e2e test to verify cpu utilization policy in appfile locally for the newly added part. Will cover it and update the doc later. Thanks for pointing it out.

Reply:
I should. My bad, I wrongly paste the vela.yaml from `examples/testapp` instead of `docs/examples/testapp` which has been fixed #569 , and I didn't do a double check. Thanks for pointing it out.

Reply:
Please simply do
```
# Number of CPU units for the service.
cpu: 256
```

not `cpuRequest` ...

Reply:
And what is the unit for this cpu limit? This seems not documented anywhere. For example, what does it mean by `cpu: 256` or `cpu: 0.5`? how user should fill this field?

Reply:
- It might not be explicit to use `cpu` as I want to emphasize it the request for cpu, and maybe later `cpuLimits` would be added. 
```
resources: {
  limits:
    cpu: parameter.cpuRequests
  requests:
    cpu: parameter.cpuRequests
}
```

Besides, it's complicated to explain the cpu unit as the definition here https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L88, while most Kubernetes users will know `cpuRequests` and `cpuLimits`.

Either way, I will add an example to the `usage` part. Thanks.
```
// +usage=CPU core requests for the workload, like `256m`(0.5 core), `1`(1 core)
	// +alias=cpu-requests
	cpuRequests?: string
```

Reply:
`cpuRequest` 解释还是不应该借助 k8s 的背景，我想想怎么改。
“k8s api resource，是 OAM/Appfile 的 schema provider，不是它的 schema 本身这个关系得明确。”

Reply:
1. 这个参数名就叫 `cpu`，不要弄那么多花样
2. CPU 单位的解释就叫：`Number of CPU units for the service.` 这里的关键是，你得让用户知道怎么填。用户是填  500m 还是 500， 还是 0.5？这个必须说清楚。

Reply:
也可以直接用 OAM spec 中的定义：https://github.com/oam-dev/spec/blob/master/core/workloads/schema/containerized_workload.md#cpu-count